cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -586,19 +594,32 @@ private ModelCompletenessRequirements modelCompletenessRequirements(Collection<G
    * @param goalNames Goal names (and empty list of names indicates all goals).
    */
   public boolean meetCompletenessRequirements(List<String> goalNames) {
-    Collection<Goal> goals = goalsByPriority(goalNames).values();
+    Collection<Goal> goals = goalsByPriority(goalNames, false).values();
     MetadataClient.ClusterAndGeneration clusterAndGeneration = _loadMonitor.refreshClusterAndGeneration();
     return goals.stream().allMatch(g -> _loadMonitor.meetCompletenessRequirements(
         clusterAndGeneration, g.clusterModelCompletenessRequirements()));
   }
 
   /**
    * Get a goals by priority based on the goal list.
+   *
+   * @param goals A list of goals to optimize.
```

Comment:
In the context of this method, are the given goals `optimized` in this method -- i.e. do we use this function to optimize the given goals as it is indicated in the description of this parameter?

Would it be more clear to just say `A list of goals`?